app/_data/docs_nav_kuma_2.10.x.yml

```diff
@@ -1,5 +1,6 @@
 generate: true
 assume_generated: true
+release: 2.10.x
```

Comment:
NIT: It's unfortunate to have to have this here when it's already in the file name. It means more places to edit when adding a new release and therefore more likeliness to get it wrong

Reply:
We can remove this in the future, it was done to make it backwards compatible with kuma who had `_dev` in the file name.